nutaudet/Target.cxx

```diff
@@ -352,6 +339,9 @@ void Target::ConstructGeometry()
 	  magField2->SetFieldValue(fField,0,0.);
 	  MagnetVol=gGeoManager->GetVolume("NudetMagnet");
 	}
+    else{
+    MagnetVol = NULL; //initialize to nothing, fixing -Wmaybe-uninitialized warnings
```

Comment:
Should be initialised directly in line 322.

Please use `nullptr`, `NULL` should not be used in modern code.

Reply:
You are right, thanks!